source/layouts/layout.slim

```diff
@@ -15,11 +15,11 @@ html.no-js lang="en"
 
   head
     meta charset="UTF-8"
+    meta name="viewport" content="width=device-width, initial-scale=1"
     == yield_content(:head)
 
     title= current_page.data.title
 
-    == stylesheet_link_tag 'application', 'normalize'
```

Comment:
Are we sure we import application in every page partial? I'm not even sure we have a page partial for every page :/

Reply:
I've confirmed that all pages are properly displayed with this. Actually I want to refactor all the stylesheets but doing so is out of scope of this PR.

Reply:
@yujinakayama ok I trust you, merge and deploy at your lesuire